docs/contributing.md

```diff
@@ -8,6 +8,8 @@ When submitting code, please make every effort to follow existing conventions
 and style in order to keep the code as readable as possible. Please also make
 sure your code compiles by running `./gradlew clean build`.
 
+When creating a pull request, please add a row in the [changelog](https://github.com/square/kotlinpoet/blob/main/docs/changelog.md) with the patch description and PR # to `Unreleased` section.
```

Comment:
supernit: I think it'd be better to move the URL to the bottom of the document, similar to the existing one, and also wrap the line to match the rest of the doc, long lines are hard to read in the IDE without horizontal scrolling.